02-Reproject-Raster-In-R.Rmd

```diff
@@ -246,15 +258,16 @@ plot(DTM_HARV,
      legend=F)
 ```
 
-We have now successfully layered the Digital Terrain Model on top of our hillshade
-to produce a nice looking, textured basemap! 
+We have now successfully layered the Digital Terrain Model on top of our
+hillshade to produce a nice looking, textured base map! 
+
+##Challenge
```

Comment:
Will you please more that into an issue in our template repo? I don't have it in my evernote account for whatever reason. It's fine if it's a big long issue! Thank you Megan.